shell/platform/glfw/flutter_glfw.cc

```diff
@@ -359,15 +374,21 @@ static void GLFWMouseButtonCallback(GLFWwindow* window,
                                     int key,
                                     int action,
                                     int mods) {
-  // Flutter currently doesn't understand other buttons, so ignore anything
-  // other than left.
-  if (key != GLFW_MOUSE_BUTTON_LEFT) {
+  FlutterPointerEvent event = {};
```

Comment:
Why was this moved? You're not using it in the new code.

Reply:
Updated.